src/Console/BaseCommand.php

```diff
@@ -45,7 +45,7 @@ abstract class BaseCommand implements CommandInterface
     public function setName(string $name)
     {
         assert(
-            strpos($name, ' '),
+            strpos($name, ' ') !== false && strpos($name, ' ') !== 0,
```

Comment:
I would rather add a `/** @phpstan-ignore-next-line */` above this line than complicating the code to please phpstan.

Reply:
Feel free to ammend, I just dont see the cost factor of current Version too high for assert calls